HotelBooking.Service/Services/BookingService.cs

```diff
@@ -13,29 +16,33 @@ public class BookingService : IBookingService
         private readonly IBookingRepository bookingRepository;
         private readonly IBookedRoom bookedRoomRepository;
         private readonly IRoomRepository roomRepository;
+        private readonly IRoomService roomServiceRepository;
 
         private readonly IMapper mapper;
         private readonly IUnitOfWork unitOfWork;
         public BookingService(IHotelRepository hotelRepository,
             IBookedRoom bookedRoomRepository, IBookingRepository bookingRepository,
-            IRoomRepository roomRepository, IMapper mapper, IUnitOfWork unitOfWork)
+            IRoomRepository roomRepository, IMapper mapper, IUnitOfWork unitOfWork,
+            IRoomService roomServiceRepository)
         {
             this.hotelRepository = hotelRepository;
             this.bookedRoomRepository = bookedRoomRepository;
             this.bookingRepository = bookingRepository;
             this.roomRepository = roomRepository;
             this.mapper = mapper;
             this.unitOfWork = unitOfWork;
+            this.roomServiceRepository = roomServiceRepository;
         }
 
-        public async Task<bool> AddBookingAsync(BookingVM model)
+        public async Task<bool> AddBookingAsync(BookingVM model,string UserId)
         {
             var booki = new Booking()
```

Comment:
what does booki mean?

Reply:
Sorry, it's my mistake. I will fix it now.